src/toggle-switch/toggle-switch.scss

```diff
@@ -16,35 +16,32 @@
     color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-2));
   }
 
-  > input {
+  > input[type='checkbox'] {
     width: 0;
     height: 0;
     appearance: none;
     opacity: 0;
     position: absolute;
 
-    // Unchecked toggle switch
+    // Toggle switch
```

Comment:
Also, if you open up 2 tabs (one with localhost your PR and one which is deployed), change the zoom and switch between those, you can see slight differences (icon placement, toggle size too).